scripts/localize_url.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+
+from pathlib import Path
+import click
+
+def get_files(path):
+  files = set()
+  for filename in Path(path).rglob('*.md'):
+    files.add(str(filename))
+  
+  return files
+
+def pprint(bad_links):
+  for k,v in bad_links.items():
+    if len(v) > 0:
+      print(k)
+      for vv in v:
+        print("--> " + vv)
+      print()
+
+def get_en_links(id_files, path):
+  links = set()
+  for f in id_files:
+    links.add(f[len(path):-3])
+  return links
+    
+def find_links(id_files, en_links, overwrite=False):
+  result = {}
+  for idf in id_files:
+    bad_links = []
+    with open(idf, "r") as f:
+      fread = f.read()
+      f.close()
+      for en_link in en_links:
+        en_link_md = "(" + en_link
+        id_link_md = "(/id" + en_link
+        if en_link_md in fread:
+          bad_links.append(en_link)
+          fread = fread.replace(en_link_md, id_link_md)
```

Comment:
what if the target doc is not translated yet?

Reply:
target docs are already translated
this because `en_links` is a list of `id_files` substring (see get_en_links function)

But I got your point, someone may pass different value to en_links

Reply:
Updated https://github.com/jk8s/sig-docs-id/pull/26/commits/d7cab4651a5e8e99dc24ed4b1e78fb3df39331e7